### PR TITLE
Fix unit test

### PR DESCRIPTION
**Pull request by siddhi-gh**
### Description
- fix unit test in `App.test.js`

Screenshot 2023-05-28 at 5 00 55 PM

### CODE REVIEW

**Review by tejas-k3 — APPROVED** [review]
Great work introducing test suite with base test case & screenshot!

File: package.json [hunk 1]
```diff
@@ -17,10 +17,10 @@
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "export SET NODE_OPTIONS=--openssl-legacy-provider && react-scripts start",
```
Comment (tejas-k3) [inline package.json:18]: Why changes are here in start?

Comment (siddhi-gh) [reply package.json:18]: To address `error:03000086:digital envelope routines::initialization` error while running App locally.

Comment (siddhi-gh) [reply package.json:18]: For more info: https://appdividend.com/2022/12/23/error-03000086-digital-envelope-routines-initialization/

**Pull request merged**